tools/README.md

```diff
@@ -0,0 +1,7 @@
+# Tools for Survey Strategy White Paper Writers
+
+This folder contains some notebooks to help support people writing cadence white papers in fall 2018. 
+
+| File | Description |
+|------|-------------|
+| [BasicNvisitsAreaTrade.ipynb](BasicNvisitsAreaTrade) | Description (Lynne!) |
```

Comment:
Need to add the other notebook, for completeness - and can you provide one-sentence descriptions please? Thanks!